code/modules/spells/artifacts.dm

```diff
@@ -150,18 +150,550 @@
+		feed_sound(user)
+		return 1
+
+/obj/item/weapon/reagent_containers/glass/cornucopia/standard_pour_into(var/mob/user, var/atom/target) // This goes into afterattack and yes, it's atom-level
+	if(!target.reagents)
+		return 0
+
+	// Ensure we don't splash beakers and similar containers.
+	if(!target.is_open_container() && istype(target, /obj/item/weapon/reagent_containers))
+		user << "<span class='notice'>\The [target] is closed.</span>"
+		return 1
+	// Otherwise don't care about splashing.
+	else if(!target.is_open_container())
+		return 0
+
+	if(!reagents || !reagents.total_volume)
+		user << "<span class='notice'>[src] is empty.</span>"
+		return 1
+
+	if(!target.reagents.get_free_space())
+		user << "<span class='notice'>[target] is full.</span>"
+		return 1
+
+	var/trans = target.reagents.add_reagent(reagents, amount_per_transfer_from_this)
+	user << "<span class='notice'>You transfer [trans] units of the solution to [target].</span>"
+	return 1
+
+
+/obj/item/enchantment/fire
+	name = "infernal enchantment rune"
+	desc = "A scroll marked with an ancient rune for infernal wrath. On the reverse side is a glyph representing a sword."
+	icon_state = "fire"
+	product = /obj/item/weapon/material/twohanded/zweihander/gotterdammerung
+	factor_paths = list(/obj/item/weapon/melee, /obj/item/weapon/material/sword, /obj/item/weapon/material/twohanded/baseballbat, /obj/item/weapon/material/twohanded/zweihander)
+
+/obj/item/weapon/material/twohanded/zweihander/gotterdammerung
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "Gotterdammerung"
+	base_icon = "flammenschwert"
+	name = "flammenschwert"
+	desc = "The mighty flamberge said to be wielded by the ancient jotunn Surtr during the end of days, Ragnarok. Its undulating blade burns bright with the flames of Muspell."
+	force = 30
+	w_class = 4.0
+	slot_flags = SLOT_BACK
+	force_wielded = 1
+	unwielded_force_divisor = 1
+	thrown_force_divisor = 0.75
+	edge = 1
+	sharp = 1
+	attack_verb = list("attacked", "chopped", "cleaved", "torn", "cut")
+	default_material = "steel"
+	wielded_ap = 60
+	unwielded_ap = 20
+	light_color = "#ED9200"
+	light_power = 1
+	light_range = 4
+
+/obj/item/weapon/material/twohanded/zweihander/gotterdammerung/process()
+	if(isliving(loc))
+		var/mob/living/M = loc
+		M.IgniteMob()
+	var/turf/location = get_turf(src)
+	if(location)
+		location.hotspot_expose(1400, 30)
+		return
+
+/obj/item/weapon/material/twohanded/zweihander/gotterdammerung/afterattack(atom/A as mob|obj|turf|area, mob/user as mob, proximity)
+	..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.apply_effect(rand(4,12),INCINERATE)
+		L.adjustFireLoss(force/2)
+
+/obj/item/enchantment/singularity
+	name = "chaos enchantment rune"
+	desc = "A scroll marked with an ancient rune for chaos. On the reverse side is a glyph representing a playing card."
+	icon_state = "singularity"
+	product = /obj/item/weapon/singularity_deck
+	factor_paths = list(/obj/item/weapon/deck, /obj/item/weapon/deck)

```

Comment:
This list seems to have two same values in it.

Reply:
Each one is distinct in its own special way.